source/testing/testing-controllers.md

```diff
@@ -30,8 +30,8 @@ export default Ember.Controller.extend({
 });
 ```
 
-`setProps` sets a property on the controller and also calls a method. In our 
-generated test, ember-cli already uses the `moduleFor` helper to setup a test 
+The `setProps` action directly sets one property, and calls the method to set the other.
```

Comment:
I originally had "a" here instead of "the" (as in the original text). I changed it at the last second because I felt it flowed better in the context of reading the page top to bottom.

What I mean is, before the code snippet of the controller, it says:
> Here we have a controller `PostsController` with two properties, a method that
> sets one of those properties, and an action named `setProps`.

So, assuming you're reading that first, you already know about _the_ method. Saying _a_ makes it sound like maybe it's another method again?

That's my logic anyway. Happy to change it to "a" if you still disagree :)